system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -167,6 +167,26 @@
                 :concept-id supplied-concept-id}
                (ingest/parse-ingest-body :xml response)))))))
 
+(deftest subscription-ingest-with-bad-query-error-test
+  (testing "ingest of a new subscription concept"
+    (let [coll1 (data-core/ingest-umm-spec-collection
+                  "PROV1"
+                  (data-umm-c/collection
+                   {:ShortName "coll1"
+                    :EntryTitle "entry-title1"})
+                  {:token "mock-echo-system-token"})
+          metadata {:CollectionConceptId (:concept-id coll1)
+                    :Query "options%5Bspatial%5D%5Bor%5D=true"}
+          concept (subscription-util/make-subscription-concept metadata)
+          response (ingest/ingest-concept
+                    concept
+                    {:accept-format :json
+                     :raw? true})
+          {:keys [errors]} (ingest/parse-ingest-body :json response)]
+      (is (re-find #"Error ingesting subscription with query parameters" (first errors)))
+      (is (re-find #"Granule search failed. status: 400 body:" (first errors))))))
```

Comment:
I meant check the exact error message that is returned from search on the validation error. As you can tell already that the whole error message is not very user friendly. The `Granule search failed. status: 400 body:" part is confusing to the end user. That is something only cmr dev cares. To the end user, the message should say something like "Subscription query failed validation with error ....".